wordpress/wp-content/plugins/memberful-wp/src/contrib/elementor.php

```diff
@@ -1,3 +1,8 @@
 <?php
 
-add_action('elementor/frontend/the_content', 'memberful_wp_protect_content');
+add_filter('elementor/frontend/builder_content_data', function($data, $post_id) {
+  if (get_queried_object_id() === $post_id) {
```

Comment:
I think that there is an edge case because `get_queried_object_id()` can return the same ID as `$post_id` even if it's not a post. There is a small chance of it happening, but do we want to ignore it?

Reply:
It looks that `get_queried_object_id()` uses `get_queried_object()` which might return one of the following: category, tag, taxonomy, posts page, single post, page, or author so in theory there might be a collision (well, I'm not sure if `builder_content_data` is called for anything else then a post).  Also, the same technique is used directly by Elementor so it's probably OK.